L1Trigger/L1THGCal/python/customClustering.py

```diff
@@ -1,232 +1,226 @@
+    parameters_c3d = histoMax_C3d_params.clone()
+    set_histomax_params(parameters_c3d, distance, nBins_R, nBins_Phi, binSumsHisto, seed_threshold)
+    process.hgcalBackEndLayer2Producer.ProcessorParameters.C3d_parameters = parameters_c3d
     return process
 
-dr_layerbylayer = ([0] + # no layer 0
-        [0.015]*7 + [0.020]*7 + [0.030]*7 + [0.040]*7 + # EM
-        [0.040]*6 + [0.050]*6 + # FH
-        [0.050]*12) # BH
 
+def custom_3dclustering_histoSecondaryMax(process,
+                                          distance=histoSecondaryMax_C3d_params.dR_multicluster,
+                                          threshold=histoSecondaryMax_C3d_params.threshold_histo_multicluster,
+                                          nBins_R=histoSecondaryMax_C3d_params.nBins_R_histo_multicluster,
+                                          nBins_Phi=histoSecondaryMax_C3d_params.nBins_Phi_histo_multicluster,
+                                          binSumsHisto=histoSecondaryMax_C3d_params.binSumsHisto,
+                                          ):
+    parameters_c3d = histoSecondaryMax_C3d_params.clone()
+    set_histomax_params(parameters_c3d, distance, nBins_R, nBins_Phi, binSumsHisto, threshold)
+    process.hgcalBackEndLayer2Producer.ProcessorParameters.C3d_parameters = parameters_c3d
+    return process
 
-dr_layerbylayer_Bcoefficient = ([0] + # no layer 0
-        [0.020]*7 + [0.020]*7 + [0.02]*7 + [0.020]*7 + # EM
-        [0.020]*6 + [0.020]*6 + # FH
-        [0.020]*12) # BH
 
 def custom_3dclustering_histoMax_variableDr(process,
-        distances = dr_layerbylayer,
-        nBins_R = 36,
-        nBins_Phi = 216,
-        binSumsHisto = binSums,                        
-        seed_threshold = 10,
-        ):
-    process = custom_3dclustering_histoMax(process, 0, nBins_R, nBins_Phi, binSumsHisto, seed_threshold)
-    parameters_c3d = process.hgcalBackEndLayer2Producer.ProcessorParameters.C3d_parameters
+                                            distances=histoMaxVariableDR_C3d_params.dR_multicluster_byLayer_coefficientA,
+                                            nBins_R=histoMaxVariableDR_C3d_params.nBins_R_histo_multicluster,
+                                            nBins_Phi=histoMaxVariableDR_C3d_params.nBins_Phi_histo_multicluster,
+                                            binSumsHisto=histoMaxVariableDR_C3d_params.binSumsHisto,
+                                            seed_threshold=histoMaxVariableDR_C3d_params.threshold_histo_multicluster,
+                                            ):
+    parameters_c3d = histoMaxVariableDR_C3d_params.clone()
+    set_histomax_params(parameters_c3d, 0, nBins_R, nBins_Phi, binSumsHisto, seed_threshold)
     parameters_c3d.dR_multicluster_byLayer_coefficientA = cms.vdouble(distances)
+    process.hgcalBackEndLayer2Producer.ProcessorParameters.C3d_parameters = parameters_c3d
     return process
 
 
+# def custom_3dclustering_histoInterpolatedMax(process,
```

Comment:
This commented function can now be removed

Reply:
done.